docs/data-clients/kubernetes.md

```diff
@@ -22,32 +21,35 @@ proxy made to apply updates very often. Skipper is used in
-  - [cookie handling](https://godoc.org/github.com/zalando/skipper/filters/cookie)
-  - [circuitbreakers](https://godoc.org/github.com/zalando/skipper/filters/circuit): consecutiveBreaker or rateBreaker
-  - [ratelimit](https://godoc.org/github.com/zalando/skipper/filters/ratelimit): based on client or backend data
-  - Shadow traffic: [tee()](https://godoc.org/github.com/zalando/skipper/filters/tee)
-- Predicates - advanced matching capability
-  - URL Path match: `Path("/foo")`
-  - Host header match: `Host("^www.example.org$")`
-  - [Querystring](https://godoc.org/github.com/zalando/skipper/predicates/query): `QueryParam("featureX")`
-  - [Cookie based](https://godoc.org/github.com/zalando/skipper/predicates/cookie): `Cookie("alpha", /^enabled$/)`
-  - [source whitelist](https://godoc.org/github.com/zalando/skipper/predicates/source): `Source("1.2.3.4/24")`
-  - [time based interval](https://godoc.org/github.com/zalando/skipper/predicates/interval)
-  - [traffic by percentage](https://godoc.org/github.com/zalando/skipper/predicates/traffic) supports also sticky sessions
+   - [collection of base http manipulations](../reference/filters.md):
+     for example [manipulating Path](../reference/filters.md#http-path), [Querystring](../reference/filters.md#http-query), [HTTP Headers](../reference/filters.md#http-headers) and [redirect](../reference/filters.md#http-redirect) handling
+   - [cookie handling](../reference/filters.md#cookie-handling)
+   - [circuitbreakers](../reference/filters.md#circuit-breakers)
+   - [ratelimit](../reference/filters.md#rate-limit): based on client or backend data
+   - [Shadow traffic filters](../reference/filters.md#shadow-traffic)
+- [Predicates](../reference/predicates.md) - advanced matching capability
+   - URL [Path](../reference/predicates.md#the-path-tree) match: `Path("/foo")`
+   - Host header match: `Host("^www.example.org$")`
+   - [Querystring](../reference/predicates.md#queryparam): `QueryParam("featureX")`
+   - [Cookie based](../reference/predicates.md#cookie): `Cookie("alpha", /^enabled$/)`
+   - [source IP allowlist](../reference/predicates.md#source): `Source("1.2.3.4/24")` or `ClientIP("1.2.3.4/24")`
+   - [time based interval](../reference/predicates.md#interval)
+   - [traffic by percentage](../reference/predicates.md#trafficsegment) supports also sticky sessions
 - Kubernetes integration
-  - All Filters and Predicates can be used with 2 annotations
-    - Predicates: `zalando.org/skipper-predicate`
-    - Filters: `zalando.org/skipper-filter`
-  - Custom routes can be defined with the annotation `zalando.org/skipper-routes`
-  - [metrics](https://godoc.org/github.com/zalando/skipper/metrics)
-  - access logs
-  - Blue-Green deployments, with another Ingress annotation `zalando.org/backend-weights`
+   - All Filters and Predicates can be used with 2 annotations
```

Comment:
We can add link about annotations here:
https://opensource.zalando.com/skipper/kubernetes/ingress-usage/#skipper-ingress-annotations

Reply:
done